tests/fixtures/valid_analysis/rules/example_rule_data_model.yml

```diff
@@ -0,0 +1,81 @@
+AnalysisType: rule
+Filename: example_rule_data_model.py
+RuleID: DataModel.BruteForceByIP
+DisplayName: Brute Force By IP
+Enabled: true
+LogTypes:
+  - OneLogin.Events
+  - GSuite.Events
+Tags:
+  - OneLogin
+  - GSuite
+Severity: Medium
+Reports:
+  MITRE ATT&CK:
+    - Brute Force:Password Spraying
+Description: A single ip address was denied access to OneLogin more times than the configured threshold.
+Threshold: 10
+DedupPeriodMinutes: 10
+Reference: https://developers.onelogin.com/api-docs/1/events/event-resource
+Runbook: Analyze the IP they came from, and other actions taken before/after. Check if a user from this ip eventually authenticated successfully.
+SummaryAttributes:
+  - account_id
+  - user_name
+  - user_id
+  - p_any_ip_addresses
+Tests:
```

Comment:
Good question, presumably for now summary attributes just won't work with udm because summary attributes get evaluated as SQL when the alert is visited, they aren't pre-calculated in the rules engine. I suppose we just generally suggest users rely on the  `p_x` fields for this.